### DISCUSSION TITLE
Can't install (macOS) installer damaged

### DISCUSSION THREAD

**Discussion by celian-hamon** [N0chteil-Productions/Apple-Music-RPC#12] [Q&A] [answered]
Cant open on m1 mac

latest version of mac os 
macbook pro m1 

<img width="324" alt="image" src="https://user-images.githubusercontent.com/61624140/157447020-be9e8c72-3a1b-4826-8b2e-5ab1bd7c2e3a.png">

AMRPC is damaged and cant be opened

**Comment by 7loud** [comment, marked as answer]
Take a look at https://github.com/ZephraCloud/Apple-Music-RPC/issues/9

**Comment by celian-hamon** [comment]
okay after disabling security it works (how do you retreive artworks from apple musics? pls can you dm me on discord cece.msi#6775 or twitter @eltriceratops pls pls )

**Comment by celian-hamon** [comment]
<img width="318" alt="image" src="https://user-images.githubusercontent.com/61624140/157520574-d2431d21-12db-4ef3-86e5-4a4235e26150.png">
error with the timestamp

**Comment by 7loud** [comment]
> <img alt="image" width="318" src="https://user-images.githubusercontent.com/61624140/157520574-d2431d21-12db-4ef3-86e5-4a4235e26150.png">
> 
> error with the timestamp

It's a Discord bug

**Comment by 7loud** [comment]
> okay after disabling security it works (how do you retreive artworks from apple musics? pls can you dm me on discord cece.msi#6775 or twitter @eltriceratops pls pls )

https://github.com/ZephraCloud/iTunes-bridge
https://github.com/ZephraCloud/Apple-Bridge
